clusterloader2/pkg/prometheus/manifests/default/prometheus-serviceMonitorKubeProxy.yaml

```diff
@@ -0,0 +1,27 @@
+{{$PROMETHEUS_SCRAPE_KUBE_PROXY := DefaultParam .PROMETHEUS_SCRAPE_KUBE_PROXY false}}
+
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: kube-proxy
+  name: kube-proxy
+  namespace: monitoring
+spec:
+  endpoints:
+    # We modify interval depending on the cluster size to avoid collecting too many samples for
+    # large clusters. Also because the tests run longer in bigger clusters we don't need to
+    # collect them as often as in smaller clusters. We use 30s interval for small clusters
+    # (# nodes <= 1000) and 1min interval in big clusters (# nodes > 1000)
+    # TODO(mm4tt): Once we prove the interval works in big clusters, simplify the expression.
+    {{if $PROMETHEUS_SCRAPE_KUBE_PROXY}}
```

Comment:
I wonder it anything won't fail if it's disabled, e.g. if there isn't a validation that endpoints list shouldn't be empty.
We'll see in the presubmit :)

Reply:
Good point. Alternatively I think we can put whole object inside this if-statement. 

I'd much rather prefer to enable this is scalability tests without changing code of clusterloader2.